boards/riscv/nrf54h20pdk_nrf54h20/nrf54h20pdk_nrf54h20_cpuppr.dts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nordic/nrf54h20_enga_cpuppr.dtsi>
+#include "nrf54h20pdk_nrf54h20-memory_map.dtsi"
+#include "nrf54h20pdk_nrf54h20-pinctrl.dtsi"
+
+/ {
+	compatible = "nordic,nrf54h20dk_nrf54h20-cpuppr";
+	model = "Nordic nRF54H20 DK nRF54H20 Peripheral Processor MCU";
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	chosen {
+		zephyr,console = &uart135;
+		zephyr,code-partition = &cpuppr_code_partition;
+		zephyr,flash = &mram1x;
+		zephyr,sram = &cpuppr_ram3x_region;
+	};
+};
+
+&cpuapp_dma_region {
+	status = "okay";
+};
+
+&grtc {
+	status = "okay";
+	owned-channels = <5>;
+};
+
+&uart135 {
+	status = "okay";
+	memory-regions = <&cpuapp_dma_region>;
```

Comment:
This property can be removed. In PPR's case, UART buffers will be placed in global RAM anyway.